cassandane/Cassandane/Cyrus/TestCase.pm

```diff
@@ -894,6 +894,125 @@ sub _need_http_tiny_env
     return 1;
 }
 
+package Cassandane::Cyrus::TestCase::User {
+  use Moo;
+
+  has username => (is => 'ro', required => 1);
+  has password => (is => 'ro', required => 1);
+  has instance => (is => 'ro', required => 1);
+
+  has _common_service_args => (
```

Comment:
Contextual detail that was lost in the move from `_setup_http_service_objects`: these args were common to _http_ services, not all services